app/src/main/res/values-in/strings.xml

```diff
@@ -0,0 +1,72 @@
+<resources>
+    <string name="app_name">Food Expiration Dates</string>
+    <string name="app_name_short">Food Expirations</string>
```

Comment:
The title of this app is translatable. If you think it's better to translate it in Indonesian, please do it.

Reply:
Thanks for correction,

I thought value of `edit_item` were not translated cause I doubt, but now, I prefer to choose to translate it kwkw 
For word `item`, it's better to translate as `item` cause common for us (Indonesian).
And the other two : `menu` and `normal` are loan language (they are correct even if they are the same as in English).

I'll update the translations, asap.

Reply:
Additional : for `app_name` and `app_name_short`, I think i'ts better to translate as originally. That is your brand and I like it 🎉